pkg/common/telemetry/names.go

```diff
@@ -2,6 +2,8 @@ package telemetry
 
 // Constants for metric keys and labels. Helps with enforcement of non-conflicting usage of same or similar names.
 // Additionally, importers of this package can get an idea of metric tags to look for.
+// While these constants are exported, it is preferable to use the functions defined in specific_metrics.go, or
+// define new such functions there
 
 const (
```

Comment:
It may be easier to understand these constants by grouping them according to function, instead of purely alphabetical.

For example, looks like you have some verbs
```
const (
  Attest
  Create
  Delete
  Fetch
  List
)
```
and some nows
```
const (
    Attestor
    Audience
)
```
then you can group functionality 
```
const (
   JWTSVIDFetch
   JWTSVIDValidate

  X509SVIDFetch // why is there no X509SVIDValidate???
)
```

combine this with suggestion for subpackages and you can have
```
package jwtsvidtelemetry
 const (
    Fetch = "fetch_jwt_svid"
    Validate = "validate_jwt_svid"
)
```

Reply:
I'm not certain how far to go with breaking down the specific labels/tags into subpackages.

@azdagron do you have a preference on this?

Reply:
my fear would be future accidental duplication or near-duplication of some labels, since it would be less common to cross-reference with labels that exist in other packages